src/app/core/teams/team-authorization.service.ts

```diff
@@ -17,7 +17,6 @@ export class TeamAuthorizationService {
 		this.sessionService
 			.getSession()
 			.pipe(
-				first(),
```

Comment:
Even once the session was being reloaded with the fix above, the team-authorization service wasn't actually listening for the updated session.  Removing the `first` operator here ensures that the service gets the latest session.